bblfsh-dashboard/values.yaml

```diff
@@ -0,0 +1,27 @@
+# Default values for bblfsh-dashboard
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: bblfsh/dashboard
+  tag: v0.1.1
```

Comment:
why? this add complexity to the user, and may end in missconfiguration since the chart is create for a release. If you take a look to the official chart, they always includes a version

Reply:
To avoid the risk of deploying an outdated version if you forget to override this. This way you'll have to update the chart every time you release a new version.